ruby/vampires.rb

```diff
@@ -0,0 +1,91 @@
+#name
+# age / where were you born
+
+# would you like some garlic bread'
+
+# would you like to enroll in the company health plan
+
+puts "How many employees will be processed?"
+number_of_employees = gets.to_i
+
+until number_of_employees == 0 
+
+  puts "Please enter your name (First Last)."
+  employee_name = gets.chomp
+
+
+  puts "Please enter your age."
+  age = gets.to_i
+
+  puts  "What year were you born (xxxx)?"
+  birth_year =  gets.to_i
+
+  if age == 2017 - birth_year 
```

Comment:
When checking something like a year consider an approach that keeps us from needing to change our code every new year.

Take a look the Ruby `Time` class.  `Time.new.year`